dev.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ea
+
+if [[ $GIT_PROTO = "ssh" ]]
+then
+  GIT_DOMAIN="git@github.com:"
+else
+  GIT_DOMAIN="https://github.com/"
+fi
+
+git clone --depth=1 --shallow-submodules --recurse-submodules ${GIT_DOMAIN}greenpeace/planet4-master-theme.git persistence/app/public/wp-content/themes/planet4-master-theme
+pushd persistence/app/public/wp-content/themes/planet4-master-theme
+npm install
+gulp git_hooks
+composer install
+popd
+
+for plugin in blocks engagingnetworks medialibrary
+do
+  git clone --depth=1 --shallow-submodules --recurse-submodules ${GIT_DOMAIN}greenpeace/planet4-plugin-${plugin}.git persistence/app/public/wp-content/plugins/planet4-plugin-${plugin}
+  pushd persistence/app/public/wp-content/plugins/planet4-plugin-${plugin}
+  npm install
+  gulp git_hooks
```

Comment:
The script failed at this line for me:
```
./dev.sh: line 14: gulp: command not found
make: *** [dev] Error 127
```

Since I don't have `gulp` available globally as a command.  Probably we should add a note about dependencies on the README for this, or maybe even adding Gulp as a depedency with `npm install --save-dev...`.

Reply:
Or maybe we can just add a warning as the one in `watch.sh`?

Reply:
We do have it as a dependency on [readme](https://github.com/greenpeace/planet4-docker-compose/blob/master/README.md#requirements).